### PR TITLE
the one that refactors lots of form things

### PR DESCRIPTION
**Pull request by sturobson**

### CODE REVIEW

**Comment by sturobson** [comment]
@khawkins98 ready for a look this PR is.

File: components/vf-form/vf-form__radio/vf-form__radio.scss [hunk 1]
```diff
@@ -16,38 +16,54 @@
...
+      filter: drop-shadow(0 0 4px set-color(vf-color--blue--light))
+    }
   }
 
   // Box focus
   &:focus + .vf-form__label::before {
     border-color: set-color(vf-color--grey--dark);
+    box-shadow: 0 0 4px set-color(vf-color--blue);
+
+    @supports (filter: drop-shadow(0 0 04px blue)) {
```
Comment (khawkins98) [inline components/vf-form/vf-form__radio/vf-form__radio.scss]: I think this is a typo, space missing between `04px`

**Comment by khawkins98** [comment]
Noticed the border on focused textfields isn't much darker, i think that's probably just an oversight. 

Normal:

image

Focused:

image


Otherwise did some cleanup on the changelog formatting and seems good.

**Comment by sturobson** [comment]
added a box shadow on the form inputs when it's hovered or focused that matches the cards.

Screenshot 2021-01-14 at 11 08 02

**Comment by khawkins98** [comment]
👍

**Review by khawkins98 — APPROVED** [review]

**Pull request merged**